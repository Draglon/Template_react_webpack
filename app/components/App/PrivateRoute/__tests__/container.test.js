import React from 'react';
import configureStore from 'redux-mock-store';
import toJson from 'enzyme-to-json';

import PrivateRouteComponent from '../container';

describe('<PrivateRouteComponent />', () => {
  const mockStore = configureStore();
  const sessionId = 'some session id';
  const state = {
    reducers: {
      login: {
        sessionId,
      },
    },
  };

  const store = mockStore(state);
  const container = shallow(<PrivateRouteComponent store={store} />);

  it('Render property', () => {
    expect(toJson(container)).toMatchSnapshot();
  });
});
