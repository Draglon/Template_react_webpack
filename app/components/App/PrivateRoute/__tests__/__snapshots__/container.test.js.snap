// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrivateRouteContainer /> Render snapshot - PrivateRouteContainer 1`] = `
<PrivateRouteComponent
  dispatch={[Function]}
  sessionId="some session id"
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
  storeSubscription={
    Subscription {
      "listeners": Object {
        "clear": [Function],
        "get": [Function],
        "notify": [Function],
        "subscribe": [Function],
      },
      "onStateChange": [Function],
      "parentSub": undefined,
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "unsubscribe": [Function],
    }
  }
/>
`;
